### ISSUE TITLE
Incompatibility with Dynamic Lights

### ISSUE THREAD

**Issue by jellysquid3** [jellysquid3/phosphor-forge#1]
[Originally reported by **BuckTheWeasel** on the Gitea repository.](https://gitea.gildedgames.com/jellysquid/Phosphor/issues/3)

>Hi
>https://minecraft.curseforge.com/projects/dynamic-lights
>this mod seems not to work with it wont display lighs when holding a lightsource
>
>Greetings Buck.

**Comment by Dimentive** [comment]
I would love to see compatibility with dynamic lights as well (including OptiFine’s dynamic lighting).

**Comment by jellysquid3** [comment]
Status update for those eagerly awaiting Dynamic Lights compatibility -- Hold tight! This issue is currently blocked by #12, but once finished I should be able to work on this issue properly.

**Comment by stormbringerGT** [comment]
Possible to get support for this issue for 1.12? Since I ditched Optifine, my players are really missing dynamic lights!

**Comment by jdawg3636** [comment]
Bump - this would be *really* nice to see 👍

**Comment by katubug** [comment]
I would also love for this to come to 1.12.2, as I'm working on a modpack with Hardcore Darkness and without Optifine. Phosphor has been great in my other pack, and I'd love to be able to use it in this one!

**Comment by xJon** [comment]
Will the mod work if it's installed server-side only, and DynamicLights is installed client-side?

**Comment by Winddbourne** [comment]
Any update on this? I see that you closed #12 without actually replacing Mixins and I'm wondering what that means for resolving this issue.

**Comment by jellysquid3** [comment]
Sorry for the lack of updates.

Support for Dynamic Lights does not _depend_ on moving away Mixins, rather it was just something I wanted to finish before adding more code. This issue has simply slipped under my radar with the work on porting Phosphor to 1.14+. I'll see what can be done for compatibility between the two mods in the upcoming days.

**Comment by Dimentive** [comment]
@jellysquid3 I’m so excited to be able to use Dynamic Lights with Phosphor.

When you finally fix whatever is causing the incompatibility with Dynamic Lights, will that fix only be for the 1.15 and/or 1.14 versions of Phosphor, or will you also fix it in the 1.12.2 version of Phosphor? I am still playing 1.12.2, so I hope you’ll also fix it for 1.12.2. Thanks for all the hard work you do! 💙

**Comment by Inf0214** [comment]
Is this still planned?

**Comment by YoItSoul** [comment]
Bump! I would love to see compatibility added.

**Comment by jellysquid3** [comment]
Phosphor for Minecraft 1.12 is no longer supported by me and will not receive additional updates outside of those for maintenance sake. This decision is the result of the greatly increased burden in supporting multiple versions of the game. It is not feasible for me as one developer to maintain 6 branches for each project (2 platforms x 3 game versions...)

All remaining issues for this version of the game will be closed and marked as "won't fix", including this one.
